deployment/document-understanding-solution.template

```diff
@@ -170,6 +170,15 @@ Resources:
               commands:
                 - echo "This buildspec is based on image - aws/codebuild/amazonlinux2-x86_64-standard:2.0"
                 - node --version
+                - echo "Installing jq package"
+                - wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+                - chmod +x ./jq
+                - cp jq /usr/bin
+                - echo "Installing pip"
+                - curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+                - python get-pip.py
+                - echo "Installing moto for running tests"
+                - pip install moto
                 - npm install -g cdk@1.18.0
```

Comment:
Seems like we have upgraded to a higher version. Could you update it?
https://github.com/awslabs/document-understanding-solution/blob/kendraMaster/source/package.json#L51

Also, just curious, is there an automated version / config file which both the yarn deploy and the cicd template can use to install the same packages and versions that are required for the install/build phase?